data_linter/schemas/config-schema.json

```diff
@@ -54,7 +54,7 @@
             "title": "The Land-base-path Schema",
             "description": "This is where data should arrive for a pipeline",
             "default": "",
-            "pattern": "^s3://",
+            "pattern": "^(s3://|/)",
```

Comment:
I would drop the pattern requirement from this schema. As I would also expect something like `land_folder/` to be allowable?

(And drop for the other paths, i.e. fail / pass / etc)